src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -0,0 +1,72 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState } from 'preact/hooks';
+import Icon from '../../Icon';
+import Instructions from '../../Instructions';
+import Donut from '../../Donut';
+import ProductListLink from '../../ProductListLink';
+import styles from './styles.scss';
+
+import { useServerData } from '../../../lib/providers';
+
+export const ShortTerm = ({ instructions, linkToProductList, disclosure, donutTimestamps, contentBodyRef }) => {
+    let qualifying;
+    let periodicPayment;
+    if (useServerData()?.views?.length > 0) {
+        qualifying = useServerData().views[0].meta.qualifying;
+        periodicPayment = useServerData().views[0].meta.periodicPayment;
+    }
+
+    const [expandedState] = useState(false);
+    return (
+        <Fragment>
+            <style>{styles._getCss()}</style>
+            <div className="content__container">
+                <main className="main">
+                    <div className="content__body" ref={contentBodyRef}>
+                        <div className="dynamic__container">
+                            <div className="content__row dynamic">
+                                <div className="content__col">
+                                    <div className="content__row donuts">
+                                        <div
+                                            className={`donuts__container ${
+                                                qualifying === 'true' ? 'donuts__qualifying' : 'donuts__non_qualifying'
+                                            }`}
+                                        >
+                                            {donutTimestamps.map((val, index) => (
+                                                <Donut
+                                                    qualifying={qualifying}
+                                                    periodicPayment={periodicPayment}
+                                                    currentNum={index + 1}
+                                                    timeStamp={donutTimestamps[index]}
+                                                >
+                                                    {qualifying &&
+                                                        `${((1 / donutTimestamps.length) * 100).toFixed(0)}%`}
+                                                </Donut>
```

Comment:
```suggestion
                                            {donutTimestamps.map((_, index) => (
                                                <Donut
                                                    qualifying={qualifying}
                                                    periodicPayment={periodicPayment}
                                                    currentNum={index + 1}
                                                    timeStamp={donutTimestamps[index]}
                                                />
```
A couple things here. Intellisense was telling me that the original first arg of the map (val) was declared but the value is never read. Since index is the second arg in order to make sure the donuts are showing the correct percentage filled up we can pass in a ```_``` to denote that the first arg isn't used. That way intellisense isn't yelling at us about val. 

Also since we no longer will be using the 25% 25% 25% 25%, we can remove the qualifying check here.